include/swift/Remote/RemoteAddress.h

```diff
@@ -59,6 +59,19 @@ class RemoteAddress {
                                  const IntegerType& rhs) {
     return lhs += rhs;
   }
+
+  template<typename IntegerType>
+  RemoteAddress& operator-=(const IntegerType& rhs) {
+    Data -= rhs;
+    return *this;
+  }
+
+  template<typename IntegerType>
+  friend RemoteAddress operator-(RemoteAddress lhs,
+                                 const IntegerType& rhs) {
+    return lhs -= rhs;
+  }
```

Comment:
Where do we use these?

Reply:
Inside `dumpAsFullMetadata`! Since I'm removing that method I'll remove this as well.